nio/src/main/scala/zio/nio/channels/SelectableChannel.scala

```diff
@@ -225,13 +225,12 @@ final class ServerSocketChannel(override protected val channel: JServerSocketCha
      * @return
      *   None if no connection is currently available to be accepted.
      */
-    def accept(implicit trace: ZTraceElement): Managed[IOException, Option[SocketChannel]] =
+    def accept(implicit trace: ZTraceElement): ZIO[Scope, IOException, Option[SocketChannel]] =
       IO.attempt(Option(channel.accept()).map(new SocketChannel(_)))
         .refineToOrDie[IOException]
-        .toManagedWith(IO.whenCase(_) { case Some(channel) =>
+        .toScoped(IO.whenCase(_) { case Some(channel) =>
           channel.close.ignore
         })
```

Comment:
Here's the first compiler error. I think I have to turn this thing inside-out or something.

Reply:
And here's the closest refactoring Rosetta Stone I could find:

https://github.com/zio/zio/pull/6420/files#diff-a99361ad81db9cffd53d642b2748a53d572c66feab37e2899eedd670279e0eefL335